roles/openshift_default_storage_class/defaults/main.yml

```diff
@@ -40,3 +40,5 @@ openshift_storageclass_default: "true"
 openshift_storageclass_name: "{{ openshift_storageclass_defaults[openshift_cloudprovider_kind]['name'] }}"
 openshift_storageclass_provisioner: "{{ openshift_storageclass_defaults[openshift_cloudprovider_kind]['provisioner'] }}"
 openshift_storageclass_parameters: "{{ openshift_storageclass_defaults[openshift_cloudprovider_kind]['parameters'] }}"
+openshift_storageclass_mount_options: []
+openshift_storageclass_reclaim_policy:
```

Comment:
Lets set this to empty string so that it would be mentioned in `defaults/main.yml`